README.md

```diff
@@ -8,44 +8,36 @@ This plugins adds a button in the options dropdown menu to start a Tour to guide
 
 You can configure some "Learn More" links that are shown in some steps (screenshare, whiteboard, general) in the settings of the plugin, see example below.
 
-## Running the Plugin From Source Code
-
-1. Start the development server:
-
-```bash
-npm install
-npm start
-```
-
-2. Add this to the `settings.yml` of the BBB HTML5-client:
-```yaml
```

Comment:
This portion seems to be core configuration of this plugin, it is not supposed to be taken off without being replaced with the newest version of this setting. @germanocaumo Could you take a look and see if it's still used in the plugin?

Reply:
Yes, it is still needed, to be able to configure the help links shown in the tour. 
@antobinary

Reply:
Done! Thanks for the feedback!